ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -67,190 +64,165 @@ const ActiveRuntimeOp = ({operation, workspaceNamespace}) => {
-                          // Show all CPU options.
-                          fp.map('cpu'),
-                          // In the event that was remove a machine type from our set of valid
-                          // configs, we want to continue to allow rendering of the value here.
-                          // Union also makes the CPU values unique.
-                          fp.union([machineType.cpu]),
-                          fp.sortBy(fp.identity)
-                        )(validLeonardoMachineTypes)}
-                        value={machineType.cpu}/>
-            </div>
-            <div>
-              <label htmlFor='runtime-ram'
-                     style={{marginRight: '.25rem'}}>RAM (GB)</label>
-              <Dropdown id='runtime-ram'
-                        disabled={true}
-                        options={fp.flow(
-                          // Show valid memory options as constrained by the currently selected CPU.
-                          fp.filter(({cpu}) => cpu === machineType.cpu),
-                          fp.map('memory'),
-                          // See above comment on CPU union.
-                          fp.union([machineType.memory]),
-                          fp.sortBy(fp.identity)
-                        )(validLeonardoMachineTypes)}
-                        value={machineType.memory}/>
-            </div>
-            <div>
-              <label htmlFor='runtime-disk'
-                     style={{marginRight: '.25rem'}}>Disk (GB)</label>
-              <InputNumber id='runtime-disk'
-                           disabled={true}
-                           showButtons
-                           decrementButtonClassName='p-button-secondary'
-                           incrementButtonClassName='p-button-secondary'
-                           value={masterDiskSize}
-                           inputStyle={{padding: '.75rem .5rem', width: '2rem'}}
-                           min={50 /* Runtime API has a minimum 50GB requirement. */}/>
-            </div>
-          </FlexRow>
-          <FlexColumn style={{marginTop: '1rem'}}>
-            <label htmlFor='runtime-compute'>Compute type</label>
-            <Dropdown id='runtime-compute'
-                      style={{width: '10rem'}}
-                      disabled={true}
-                      options={['Dataproc cluster', 'Standard VM']}
-                      value={isDataproc ? 'Dataproc cluster' : 'Standard VM'}/>
-          </FlexColumn>
-        </div>
-        <FlexRow style={{justifyContent: 'flex-end', marginTop: '.75rem'}}>
-          <Button disabled={true}>Create</Button>
-        </FlexRow>
-        {activeRuntimeOp && <React.Fragment>
-          <hr/>
-          <ActiveRuntimeOp operation={activeRuntimeOp.operation} workspaceNamespace={workspace.namespace}/>
-        </React.Fragment>}
-      </div>;
-    }
+      >{currentRuntime ? 'Update' : 'Create'}</Button>
+    </FlexRow>
+    {activeRuntimeOp && <React.Fragment>
```

Comment:
Could you drop this for now? It is flashing in and out of existence on the page, which is quite jarring.